diff --git a/ios/DevMenuViewController.swift b/ios/DevMenuViewController.swift
index 1234567..abcdefg 100644
--- a/ios/DevMenuViewController.swift
+++ b/ios/DevMenuViewController.swift
@@ -63,7 +63,11 @@ open class DevMenuViewController: UIViewController {
   // MARK: private

   private func initialProps() -> [String: Any] {
-    let isSimulator = TARGET_IPHONE_SIMULATOR > 0
+    #if targetEnvironment(simulator)
+    let isSimulator = true
+    #else
+    let isSimulator = false
+    #endif

     return [
       "showOnboardingView": manager.shouldShowOnboarding(),
